ui2/src/pages/clearing/Members.tsx

```diff
@@ -115,6 +131,51 @@ const ClearingMembersComponent: React.FC<RouteComponentProps & ServicePageProps<
           };
         })}
       />
+      <Header as="h2">Cleared Trades</Header>
+      <StripedTable
+        loading={clearedTradesLoading}
+        rowsClickable
```

Comment:
I don't think `rowsClickable` needs to be here - unless u provide an `onClick` function (plus, there is a button in the row data)